packages/client-sdk-nodejs/src/internal/internal-auth-client.ts

```diff
@@ -28,18 +29,38 @@ import {
   AllTopics,
   isCacheName,
   isTopicName,
+  GenerateDisposableToken,
+  AllItems,
+  isCacheItemKey,
+  isCacheItemKeyPrefix,
 } from '@gomomento/sdk-core';
 import {IAuthClient} from '@gomomento/sdk-core/dist/src/internal/clients';
 import {AuthClientProps} from '../auth-client-props';
 import {normalizeSdkError} from '@gomomento/sdk-core/dist/src/errors';
 import {
+  TemporaryTokenScope,
   asCachePermission,
   asPermissionsObject,
   asTopicPermission,
   isCachePermission,
   isPermissionsObject,
   isTopicPermission,
+  PredefinedScope,
+  isTemporaryTokenPermissionsObject,
+  asTemporaryTokenPermissionsObject,
+  TemporaryTokenCachePermission,
+  isTemporaryTokenCachePermission,
+  asTemporaryTokenCachePermission,
 } from '@gomomento/sdk-core/dist/src/auth/tokens/token-scope';
+import {permission_messages} from '@gomomento/generated-types/dist/permissionmessages';
```

Comment:
Permissions stuff moved to a different proto package so there are a lot of edits related to fixing the path to those types